nsfw/lang/de/messages.po

```diff
@@ -26,13 +27,13 @@ msgstr "Not Safe for Work (Filter für ungewollte Inhalte) Einstellungen:"
 
 #: nsfw.php:80
```

Comment:
This isn't right, there is no translated string at this line anymore.

Reply:
The version of the PHP file from which transifex has the messages.po file had the string in that line. I hope that transifex will soon pick up the new source files, just remembered that you had changed the path from master to develop and transifex was not updated to that information.